### DISCUSSION TITLE
Supervision Command class issue

### DISCUSSION THREAD

**Discussion by mustafayuce33** [zwave-js/zwave-js#6767] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
While I am adding "AEOTEC NanoMote Quad Z-wave Plus" device I encountered a problem with group lifeline. Lifeline group can not be added with this library. The problem seems to be related with supervision command class used in AssociationCCSet. If I try to see association without supervision then everything is fine. Otherwise it is problematic. 

What did you expect to happen instead?
I except that group lifeline is added during interview

  1. Try to add "AEOTEC NanoMote Quad Z-wave Plus"
  2. See if group lifeline is added or not

### Software versions

Driver (node-zwave-js): 12.5.4

### Device information

Manufacturer: ...
Model name: ...
Node ID: ...

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment]
> * I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

Where is that?
